david_vicklund/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "david_vicklund",
+  "version": "0.1.0",
+  "description": "Short greeting server",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "gulp-jshint": "~1.11.2",
+    "gulp-mocha": "~2.1.3"
```

Comment:
you have these two as dependencies as well as dev-dependencies. probalby want to delete these lines under dependencies.